dp/jobqueueing/alembic/env.py

```diff
@@ -0,0 +1,105 @@
+"""
+Customized alembic environment manager.
+
+This manager enables command-line specification of the database to migrate.
+This simplifies testing and use in various environments, e.g., local dev machine vs. compute node
+(with test and prod databases on compute node).
+
+The database to migrate is specified using the command-line argument `-x`, as follows::
+
+    alembic -x db=<db-name> upgrade ...
+
+This syntax is now required; i.e., `-x db=<db-name>` cannot be omitted from the command.
+
+The `<db-name>` section must appear in `alembic.ini`, with a `sqlalchemy.url =` line.
+For example, to use `alembic -x db=test upgrade ...`::
+
+    [test]
+    sqlalchemy.url = sqlite:////path/to/database/test.sqlite
+
+Adapted from https://gist.github.com/twolfson/4bc5813b022178bd7034
+"""
+from __future__ import with_statement
```

Comment:
This is generic code that comes with Alembic. We can modify it, but I usually prefer to leave what amounts to the source of an imported package alone.